src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -494,8 +493,8 @@ ucs_status_t uct_rc_verbs_ep_flush(uct_ep_h tl_ep, unsigned flags,
     uct_rc_verbs_ep_t *ep = ucs_derived_of(tl_ep, uct_rc_verbs_ep_t);
     ucs_status_t status;
 
-    if (comp != NULL) {
-        return UCS_ERR_UNSUPPORTED;
+    if (!uct_rc_iface_has_tx_resources(&iface->super)) {
```

Comment:
I see. But is this situation possible? I mean that you can't add anything to pending while you have send resources. From the other side as soon as some operation completed it will trigger processing all pendings if any. So, how can we have pendings while all resources are available?